vnext/Microsoft.ReactNative/ABICxxModule.h

```diff
@@ -45,4 +45,6 @@ struct ABICxxModule : facebook::xplat::module::CxxModule {
   std::vector<ConstantProvider> m_constants;
 };
 
+constexpr auto DefaultEventEmitterName = "RCTDeviceEventEmitter";
```

Comment:
>DefaultEventEmitterName [](start = 15, length = 23)

I wonder why want it to be default?
The FB docs saying that this is a deprecated event emitter and must not be used for new modules.
It would be better if we follow the best practices by default.

Reply:
Where are you seeing that? Every doc and every public native module uses it.

Reply:
https://stackoverflow.com/questions/36692416/deviceeventemitter-vs-nativeappeventemitter